### PR TITLE
update to zend-expressive 1.0.0

### PR DESCRIPTION
**Pull request by sandrokeil**
Can be merged if `prooph/psr7-middleware` 0.1.0 has been released.

* [x] update dependencies
* [x] move config files to default location
* [x] update command routes for prooph PSR-7 middleware
* [x] update build scripts
* [x] update command handler

### CODE REVIEW

**Comment by prolic** [comment]
fantastic! :+1:

**Comment by prolic** [comment]
@codeliner can you add prooph/psr7-middlewar to packagist an rerun the tests, please?

**Comment by prolic** [comment]
@sandrokeil does this PR also solve https://github.com/prooph/proophessor-do/issues/11 ?

File: src/Middleware/JsonError.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+namespace Prooph\ProophessorDo\Middleware;
+
+use Psr\Http\Message\ResponseInterface as Response;
+use Psr\Http\Message\ServerRequestInterface as Request;
+use Zend\Stratigility\ErrorMiddlewareInterface;
+
+/**
+ * Error middleware to handle json requests
+ */
+class JsonError implements ErrorMiddlewareInterface
```
Comment (sandrokeil) [inline src/Middleware/JsonError.php]: @prolic Yes it should resolve #11 with this JsonError middleware. /cc @codeliner

**Comment by codeliner** [comment]
great job!

**Pull request merged**